resources/views/dash.blade.php

```diff
@@ -20,74 +23,128 @@
+      @endif
+    </div>
+
+    <div class="content-section" id="section1">
+      <!-- This user's posts -->
+      <div>
+        <div>
+            @if(count($tattoos) == 0)
+              <span>No posts to display.</span>
+            @endif
+            @foreach($tattoos as $tattoo)
+              <div class="card mb-2">
+              <img src="{{Helpers::getUsersTattoos($tattoo, $user)}}" width="250px" height="250px" />
+                <p>{{ $tattoo->getDescription() }}</p>
+              </div>
+            @endforeach
+            {{$tattoos->links()}} <!--Links to another subpage if there are more than the paginated tattoos -->
+        </div>
+      </div>
     </div>
-  <!-- This user's posts -->
-  <div>
-    <div>
-        @if(count($tattoos) == 0)
-          <span>No posts to display.</span>
-        @endif
-        @foreach($tattoos as $tattoo)
-          <div class="card mb-2">
-          <img src="{{Helpers::getUsersTattoos($tattoo, $user)}}" width="250px" height="250px" />
-            <p>{{ $tattoo->getDescription() }}</p>
-          </div>
-        @endforeach
-        {{$tattoos->links()}} <!--Links to another subpage if there are more than the paginated tattoos -->
+
+    <div class="content-section" id="section2">
+      <!--If the user logged in is the same user that owns the profile, display profile customization features -->
+
+      @if(Auth::user()->id == $user->id)
+        <div>
+          <form enctype="multipart/form-data" action="{{route('dash.update_profile', ['id' => $user->getDisplayName()])}}" method="POST">
+            @csrf
+            <label>Change Photo</label>
+            <input type="file" name="avatar"></input>
+            <br>
+
+            <label>Change Name</label>
+            <input type="text" name="name" value="{{$user->getName()}}"></input>
+            <br>
+
+            <label>Change Username</label>
+            <input type="text" name="new_display_name" value="{{$user->getDisplayName()}}"></input>
+            <br>
+
+            <span>Change Pronouns</span>
+            <input type="text" name="pronouns" value="{{$user->getPronouns()}}"></input>
+            <br>
+
+            <span>Change Bio</span>
+            <input type="text" name="bio" value="{{$user->getBio()}}"></input>
+            <br>
+
+            <span>Change Age</span>
+            <input type="text" name="age" value="{{$user->getAge()}}"></input>
+            <br>
+
+            <label>Change Virgin Status</label>
+            <select name="virgin_status">
+              <option value="Virgin">Virgin</option>
+              <option value="Non virgin">Not a Virgin</option>
+              <option vlaue="N/A">N/A</option>
+            </select>
+            <br>
+            
+            <input type="submit" class="btn-dark"></input>
+          </form>
+        </div>
+      @endif
     </div>
   </div>
 
+  <script type="text/javascript">
```

Comment:
Yeah as we talked about before, we should try to put this in its own script file and reference the source of it in a @section('scripts'). There's an example of this is in /resources/views/tattoo/upload.blade.php
scripts are in public/js/